libraries/hatch-server-user-management/src/AWSCognitoClient.ts

```diff
@@ -224,4 +243,40 @@ export default class AWSCognitoClient implements UserServiceClient {
     return pemCerts;
   }
   
+  private convertAWSErrorToUserManagementError(awsError: AWSError) {
+    switch (awsError.code) {
```

Comment:
I think you'll also need `InvalidParameterException` for forgot password. This will be sent if forgot password is issued for an account that has not yet been confirmed.

Reply:
That is true for the ForgotPassword API but we are using the AdminResetUserPassword API and that doesn't seem to have that problem